### PR TITLE
Change non-ASCII quotes to ASCII quotes

### LINKED ISSUE #1130: Change non-ASCII quotes “ to ASCII quotes "
The following files use non-ASCII double quotes `“”`, but ideally we should use ASCII double quotes `""`:
```
examples/gallery/lines/linestyles.py:Giving “dotted” or “.” yields a dotted line, whereas a dashed pen is requested
examples/gallery/lines/linestyles.py:with “dashed” or “-”. Also combinations of dots and dashes, like “.-” for a
examples/projections/misc/misc_robinson.py:the world map “look” right 22. The scale is true along latitudes 38. The projection was
examples/projections/misc/misc_sinusoidal.py:“Equal-area Mercator” projection. The central meridian is a straight line; all other
examples/projections/misc/misc_winkel_tripel.py:German word “tripel” refers to this junction of where each of these elements are least
examples/projections/misc/misc_winkel_tripel.py:and Son started to use it in its world-renowned “The Times Atlas of the World” in the
pygmt/src/coast.py:        Select filling or clipping of “dry” areas.
pygmt/src/coast.py:        Select filling or clipping of “wet” areas.
pygmt/src/grdtrack.py:    ``interpolation``; Default uses “natural” conditions (second partial
```

### PR DESCRIPTION
**Pull request by shahid-0**
**Description of proposed changes**

Fixes #1130 

**Reminders**

- [x] Run `make format` and `make check` to make sure the code follows the style guide.
- [x] Add tests for new features or tests that would have caught the bug that you're fixing.
- [x] Add new public functions/methods/classes to `doc/api/index.rst`.
- [x] Write detailed docstrings for all functions/methods.
- [x] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

**Comment by core-man** [comment]
I think you should directly revise all the original python files instead of creating a new `ascii.py` file. E.g.,

https://github.com/GenericMappingTools/pygmt/blob/a74fd396c09ee4d666edbfc1a56f020fd5983e1f/examples/gallery/lines/linestyles.py#L11

Revise this line to be:
```Pthon
Giving "dotted" or "." yields a dotted line, whereas a dashed pen is requested
```

**Comment by shahid-0** [comment]
> I think you should directly revise all the original python files instead of creating a new `ascii.py` file. E.g.,
> 
> https://github.com/GenericMappingTools/pygmt/blob/a74fd396c09ee4d666edbfc1a56f020fd5983e1f/examples/gallery/lines/linestyles.py#L11
> 
> This line should be revised to be:
> 
> ```python
> Giving "dotted" or "." yields a dotted line, whereas a dashed pen is requested
> ```

Sorry for the mistake, I am a beginner I think that's why I make mistake I try to check all python files.

**Comment by core-man** [comment]
Great~ Please make similar revisions for other python files.

**Comment by seisman** [comment]
@shahid-0 Thanks for your contribution. Please make all changes in one single branch and pull request, instead of creating many pull requests like #1141.

**Comment by seisman** [comment]
Ping @shahid-0 just in case you forget this PR.

**Comment by shahid-0** [comment]
> Ping @shahid-0 just in case you forget this PR.
sorry for this I am increasing my python skills, so that's why I am not here. In some days I will again start contributing.

File: ascii.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+examples/gallery/lines/linestyles.py:Giving "dotted" or "." yields a dotted line, whereas a dashed pen is requested
```
Comment (seisman) [inline ascii.py:1]: Please remove this file.

**Comment by seisman** [comment]
The issue has been fixed by another PR #1179. Closing.

**Pull request closed without merging**